src/featureFlags.js

```diff
@@ -6,10 +6,16 @@ import {
   UNLEASH_URL,
   UNLEASH_CLIENT_KEY,
   UNLEASH_POLLING_INTERVAL,
+  STAGE,
 } from './constants';
 
 export const Events = {
   WALLET_SERVICE_ENABLED: 'wallet-service-enabled',
+  PUSH_NOTIFICATION_ENABLED: 'push-notification-enabled',
+};
+
+const UnleashEvent = {
+  UPDATE: 'update',
 };
 
 export class FeatureFlags extends events.EventEmitter {
```

Comment:
Since we are refactoring this class, should we transform it into a saga? Maybe it can export an event channel so other sagas can listen to state updates

Reply:
It's a good idea, but I believe it should be a design. This way we can devise a better mechanism.